.prettierrc

```diff
@@ -2,5 +2,7 @@
   "$schema": "http://json.schemastore.org/prettierrc",
   "arrowParens": "avoid",
   "endOfLine": "auto",
+  "semi": true,
+  "printWidth": 120,
```

Comment:
```suggestion
```

This change is going to affect 10K+ files, and should be done at once. It should be done when there are very few open PRs to avoid conflicts.

There's already a existing PR which is pending review from @AllanFly120 https://github.com/aws/aws-sdk-js-v3/pull/1185
Once approved, the last two commits would be removed and recreated after rebase.

Reply:
This affected 58 files, I reverted the commit regardless